src/composables/useIsInLobby.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * @copyright Copyright (c) 2024, Dorra Jaouad <dorra.jaoued1@gmail.com>
+ *
+ * @author Dorra Jaouad <dorra.jaoued1@gmail.com>
+ * @author Daniel Calviño Sánchez <danxuliu@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { computed } from 'vue'
+
+import { useStore } from './useStore.js'
+import { PARTICIPANT, WEBINAR } from '../constants.js'
+
+/**
+ *
+ */
+export function useIsInLobby() {
+	const store = useStore()
+	const token = computed(() => store.getters.getToken())
+	const conversation = computed(() => store.getters.conversation(token.value))
```

Comment:
If the only work of this composable is to calculate some values based on a store state, then it should be 3 getters in the store.

It doesn't work with a component instance context, doesn't have any params.

So we don't need to calculate a new copy of these values in each instance as we base on global state anyway.

Reply:
Moved to conversations store.